crates/dpapi/src/blob.rs

```diff
@@ -173,18 +170,18 @@ impl DpapiBlob {
                     unprotected_attrs: Optional::from(None),
                 })?)),
             },
-            &mut writer,
+            &mut *dst,
```

Comment:
suggestion: It looks like this could be this:
```suggestion
            dst,
```